src/main/java/frc/robot/commands/RotatorToPosition.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.subsystems.RotatorSubsystem;
+import frc.robot.subsystems.TelescopeSubsystem;
+
+public class RotatorToPosition extends CommandBase {
+    private final RotatorSubsystem rotatorSubsystem;
+    private final TelescopeSubsystem telescope;
+    private final double setpoint;
+    private final TelescopeToPosition resetTelescope;
+    private boolean resettingElevator;
+    public RotatorToPosition(RotatorSubsystem rotatorSubsystem, TelescopeSubsystem telescope, double setpoint)
+    {
+        this.rotatorSubsystem = rotatorSubsystem;
+        this.setpoint = setpoint;
+        this.telescope = telescope;
+        addRequirements(rotatorSubsystem);
+
+        resetTelescope = new TelescopeToPosition(telescope, 0);
+    }
+    @Override
+    public void initialize() {
+        if (telescope.currentTicksToPercent() > telescope.ticksToPercent(-2048) && Math.abs(telescope.getCurrentPosition() - setpoint) > telescope.ticksToPercent(-4096)){
```

Comment:
Hrm, this doesn't feel much different. There's still -2048, -4096. I'd have expected something like `telescope.currentPosition > 0.2`, i.e., "is the telescope more than 20% extended?"

Reply:
riiiiiight okay that makes sense, fixing!

Reply:
So, I like percent for general interfacing. For some applications, I will need to use ticks. I have gone through and replaced *most* times that the user sets the position of the telescope, such as in `TelescopeToPosition.java` to being a percent. However, in some places, I am keeping it as ticks *for now*. 

After GPK I will feel more comfortable doing a bigger general refactor.